Sources/Contour/Configurator/EndpointElement.cs

```diff
@@ -175,5 +177,21 @@ public ValidatorCollection Validators
             }
         }
 
+        [ConfigurationProperty("excluded-headers")]
+        public string ExcludedHeadersString => (string)base["excluded-headers"];
+
+        public IEnumerable<string> ExcludedHeaders
```

Comment:
It's only reading properties. No extra operations. String operations are required to manage serialization used in this config section.